client.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python
+
```

Comment:
The filename isn't significant, what does `client.py` mean? It is very generic. Please rename it to something which resembles the content of the script